cclient/serialization/json/json_serializer.c

```diff
@@ -490,6 +490,64 @@ static retcode_t hash8019_stack_to_json_array(hash8019_stack_t stack,
   return RC_OK;
 }
 
+static retcode_t hash8019_array_to_json_array(hash8019_array_p array,
+                                              cJSON* const json_root,
+                                              char const* const obj_name) {
+  size_t array_count = 0;
+  cJSON* array_obj = NULL;
+  trit_t trytes_out[NUM_TRYTES_SERIALIZED_TRANSACTION + 1];
```

Comment:
Should be `tryte_t`

Reply:
And do we really need the `+ 1` here ?

Reply:
The ending char is required by cJSON_CreateString.